modules/installation-and-upgrade/pages/container-deployment/mlm/server-migration-mlm.adoc

```diff
@@ -181,25 +181,25 @@ For example on {productname} server:
 mgradm migrate podman <oldserver.fqdn> --prepare
 ----
 ====
-+
+
 .Procedure: Final Migration
-.. Stop the {productname} services on 4.3 Server:
+. Stop the {productname} services on 4.3 Server:
 +
 ----
 spacewalk-service stop
 ----
-.. Stop the PostgreSQL service on 4.3 Server:
+. Stop the PostgreSQL service on 4.3 Server:
 +
 ----
 systemctl stop postgresql
 ----
-.. Perform the migration on {productname} server
+. Perform the migration on {productname} server
```

Comment:
We need to check if rendering would be fine here. I added second level of navigation as with 1st it was looking bit strange but then I didn't check the rendered version. I only checked with atom asciidoc plugin.